internal/cli/job_list.go

```diff
@@ -276,6 +302,24 @@ func (c *JobListCommand) Flags() *flag.Sets {
 				"Can be repeated multiple times.",
 		})
 
+		f.StringVar(&flag.StringVar{
+			Name:   "pipeline-id",
+			Target: &c.flagPipelineId,
+			Usage:  "List jobs initiated by the specific pipeline, look up by pipeline ID.",
+		})
+
+		f.StringVar(&flag.StringVar{
+			Name:   "pipeline-name",
+			Target: &c.flagPipelineName,
+			Usage:  "List jobs initiated by the specific pipeline, look up by pipeline owner.",
+		})
+
+		f.IntVar(&flag.IntVar{
+			Name:   "run",
```

Comment:
To stick with the convention, should this be `"pipeline-run"`? I think that makes sense since this is off of `waypoint job list`.

Reply:
Oh I was thinking since it would be redundant to have `-pipeline-run` since `-run` must be used together with `-pipeline`, so `waypoint job list -pipeline=x -run=y` is cleaner than `waypoint job list -pipeline=x -pipeline-run=y`